src/ResourceManager/Compute/Commands.Compute/Extension/AEM/SetAzureRmVMAEMExtension.cs

```diff
@@ -179,45 +179,58 @@ public override void ExecuteCmdlet()
 
                 // Get Disks
                 var accounts = new Dictionary<string, string>(StringComparer.InvariantCultureIgnoreCase);
-                var accountName = this._Helper.GetStorageAccountFromUri(osdisk.Vhd.Uri);
-                var storageKey = this._Helper.GetAzureStorageKeyFromCache(accountName);
-                accounts.Add(accountName, storageKey);
+                if (osdisk.ManagedDisk == null)
```

Comment:
Also, need to add changes to the Changelog for compuete, in the 'Current Version' section here: https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/Compute/ChangeLog.md